apps/src/sites/code.org/pages/public/learn/local.js

```diff
@@ -117,31 +127,189 @@ function loadMap(locations) {
 
   // Reset the map.
   $('#gmap').html('');
-  gmap = new Maplace();
-
-  var map_options = {
-    map_options: {
-      set_center: [lat, lng],
-      zoom: 12
-    },
-    controls_type: 'list',
-    controls_on_map: false
-  };
+  if (window.location.search.includes('mapbox')) {
+    const featureList = locations.map((location, index) => ({
+      type: 'Feature',
+      properties: {
+        description: location.html,
+        title: location.title,
+        index
+      },
+      geometry: {
+        type: 'Point',
+        coordinates: [parseFloat(location.lon), parseFloat(location.lat)]
+      }
+    }));
+    gmap = new mapboxgl.Map({
+      container: 'gmap',
+      style: 'mapbox://styles/mapbox/streets-v11',
+      zoom: 10,
+      minZoom: 1,
+      center: [lng, lat]
+    });
+    gmap.on('load', function() {
+      gmap.loadImage(
+        'https://docs.mapbox.com/mapbox-gl-js/assets/custom_marker.png',
+        function(error, image) {
+          // do stuff
+          if (error) {
+            throw error;
```

Comment:
Can we log this error in NewRelic or something? It would definitely be good to know if we're getting errors in these maps

Reply:
Brendan suggested we punt this to a future PR across all implementations, so don't worry about this for this PR.